tests/fox_scraper_tests.py

```diff
@@ -1,12 +1,81 @@
 from nose.tools import *
-import fox_scraper
+from fox_scraper import Scraper
+from fox_scraper import FoxItem
+from bs4 import BeautifulSoup
+from tests.items_src import catalog
 
```

Comment:
Next time, it won't be an excuse to why used method a and not b.
For now, http://pythontesting.net/framework/nose/nose-introduction/#example here you have an example of unitests in a class instead of functions
http://ivory.idyll.org/articles/nose-intro.html this another nice intro on nose